src/xmipp/libraries/interface/docfile.h

```diff
@@ -86,6 +86,9 @@ class DocLine
      */
     DocLine(const DocLine& line);
 
+    /** Destructor */
+    ~DocLine() {}
```

Comment:
probably same problem as above

Reply:
The empty constructor is not trivial. It has to be there.

Reply:
Same here - simply remove copy assignment operator and copy constructor, instead of adding (default) destructor

Reply:
Done